src/main/java/dev/jbang/cli/Export.java

```diff
@@ -22,6 +31,105 @@
 	}, description = "Force export, i.e. overwrite exported file if already exists", defaultValue = "false")
 	boolean force;
 
+	@CommandLine.Option(names = {
+			"--style" }, description = "Which style should the export be. Valid values: ${COMPLETION-CANDIDATES}", defaultValue = "local")
+	Style style;
+
+	enum Style {
+		local {
+
+			public int apply(Export export, Script script, Path outputPath) throws IOException {
+				// Copy the JAR or native binary
+				Path source = script.getJar().toPath();
+				if (export.nativeImage) {
+					source = getImageName(source.toFile()).toPath();
+				}
+
+				if (outputPath.toFile().exists()) {
+					if (export.force) {
+						outputPath.toFile().delete();
+					} else {
+						export.warn("Cannot export as " + outputPath + " already exists. Use --force to overwrite.");
+						return EXIT_INVALID_INPUT;
+					}
+				}
+
+				Files.copy(source, outputPath);
+				export.info("Exported to " + outputPath);
+				return EXIT_OK;
+			}
+		},
+		slim {
+			@Override
+			public int apply(Export export, Script script, Path outputPath) throws IOException {
+				// Copy the JAR or native binary
+				Path source = script.getJar().toPath();
+				if (export.nativeImage) {
+					source = getImageName(source.toFile()).toPath();
+				}
+
+				if (outputPath.toFile().exists()) {
+					if (export.force) {
+						outputPath.toFile().delete();
+					} else {
+						export.warn("Cannot export as " + outputPath + " already exists. Use --force to overwrite.");
+						return EXIT_INVALID_INPUT;
+					}
+				}
+
+				File tempManifest;
+
+				Files.copy(source, outputPath);
+				if (!export.nativeImage) {
+					try (JarFile jf = new JarFile(outputPath.toFile())) {
+						String cp = jf.getManifest().getMainAttributes().getValue(Attributes.Name.CLASS_PATH);
+						String[] jars = cp.split(" ");
+						File libsdir = new File(outputPath.getParent().toFile(), "libs");
+						if (jars.length > 0) {
+							if (!libsdir.exists()) {
+								libsdir.mkdirs();
+							}
+						}
+						StringBuffer newPath = new StringBuffer();
+						for (String jar : jars) {
+							Path file = downloadFile(new File(jar).toURI().toString(), libsdir);
+							newPath.append(" libs/" + file.toFile().getName());
+						}
+
+						Path tempDirectory = Files.createTempDirectory("jbang-export");
+						tempManifest = new File(tempDirectory.toFile(), "MANIFEST.MF");
+						jf.getManifest().getMainAttributes().put(Attributes.Name.CLASS_PATH, newPath.toString());
+						// System.out.println("writing manifest " + jf.getManifest() + " to " +
+						// tempManifest);
+						jf.getManifest().write(new FileOutputStream(tempManifest));
+					}
+
+					List<String> optionList = new ArrayList<String>();
+					optionList.add("jar"); // TODO locate it on path ?
+					optionList.add("ufm");
```

Comment:
Why are you using the `jar` command?

Reply:
If we really need to use it I'd at least use the method that resolves commands against the active Jdk : `BaseBuildCommand.resolveInJavaHome()`

Reply:
> Why are you using the `jar` command?

fastest way I could find to modify a jar "in place".

and yes, should make it use similar lookup in javahome

Reply:
But we're already generating JAR files with MANIFESTs without using the `jar` command, right? Can't we somehow re-use that?

Reply:
we could, but then it would need to be done at build time, not export as the code in build is about creating a full new jar.

Reply:
Ok, but using similar code to update an existing JAR should be possible too, right?

Reply:
yes, but it would be quite alot more verbose.